.pylintrc

```diff
@@ -107,7 +107,52 @@ ignore-imports=yes
 
 [MESSAGES CONTROL]
 
-disable=consider-using-ternary,locally-disabled,locally-enabled,logging-not-lazy,abstract-method,arguments-differ,broad-except,duplicate-code,fixme,inconsistent-return-statements,len-as-condition,missing-raises-doc,multiple-constructor-doc,no-else-return,no-member,no-self-use,not-context-manager,redefined-variable-type,redundant-returns-doc,too-many-arguments,too-many-boolean-expressions,too-many-branches,too-many-instance-attributes,too-many-lines,too-many-locals,too-many-public-methods,too-many-nested-blocks,too-many-statements,wrong-import-order
+disable=abstract-method,
+        arguments-differ,
+        broad-except,
+        consider-using-ternary,
+        duplicate-code,
+        fixme,
+        inconsistent-return-statements,
+        len-as-condition,
+        locally-disabled,
+        locally-enabled,
+        logging-not-lazy,
+        multiple-constructor-doc,
+        no-else-return,
+        no-member,
+        no-self-use,
+        not-context-manager,
+        redefined-variable-type,
+        redundant-returns-doc,
+        too-many-arguments,
+        too-many-boolean-expressions,
+        too-many-branches,
+        too-many-instance-attributes,
+        too-many-lines,
+        too-many-locals,
+        too-many-nested-blocks,
+        too-many-public-methods,
+        too-many-statements,
+        wrong-import-order,
+# TODO(#12912): Remove these after the Python 3 migration.
+        arg-name-for-non-keyword-arg,
+        arguments-renamed,
+        consider-using-dict-items,
+        consider-using-with,
+        cyclic-import,
+        deprecated-method,
+        import-outside-toplevel,
+        missing-raises-doc,
```

Comment:
I think this is something that we might want to reintroduce since I think that having raises in doc is quite useful and I didn't find any explanation on why is this disabled.